pkg/chunk/cache/memcached_client_selector_test.go

```diff
@@ -0,0 +1,40 @@
+package cache_test
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/bradfitz/gomemcache/memcache"
+	"github.com/cortexproject/cortex/pkg/chunk/cache"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMemcachedJumpHashSelector_PickSever(t *testing.T) {
+	s := cache.MemcachedJumpHashSelector{}
+	err := s.SetServers("google.com:80", "microsoft.com:80", "duckduckgo.com:80")
+	require.NoError(t, err)
+
+	distribution := make(map[net.Addr]int)
+
+	for i := 0; i < 100; i++ {
+		key := fmt.Sprintf("key-%d", i)
+		addr, err := s.PickServer(key)
+		if assert.NoError(t, err) {
```

Comment:
Why did we switch to `assert` here. I think we use `require` everywhere?

Reply:
I switched to `assert` just because I didn't think in this case the test needed to halt at the first error. I'll change it to `require`.